engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -953,6 +963,109 @@ class RuntimeServerTest
     )
   }
 
+  it should "send method pointer updates of partially applied type method returning a method" in {
```

Comment:
What will happen when the result is a function and remains a function yet its schema changes on subsequent evaluation of the result? Will we deliver an update to value/schema? 

I am envisioning following test case:
```haskell
f a b c = a + b + c

fn what = case what of
    0 -> f
    1 -> f 3
    2 -> f 3 8
    _ -> f 3 8 9
```
what will happen if some code invoke `fn 1` first and then the code get changed and invokes `fn 2` - will the IDE be notified about the schema change?